packages/ledger.rb

```diff
@@ -9,19 +9,6 @@ class Ledger < Package
   source_url 'https://github.com/ledger/ledger/archive/v3.1.3.tar.gz'
```

Comment:
Why not update too?  https://github.com/ledger/ledger/archive/refs/tags/v3.2.1.tar.gz

Reply:
Or even better, why not just use a git tag instead of an archive

Reply:
@supechicken: This PR is still in Draft mode.  Were you still testing?